src/components/SecondaryPanes/BreakpointItem.js

```diff
@@ -30,6 +34,41 @@ function getBreakpointLocation(source, line, column) {
 }
 
 class BreakpointItem extends Component<Props> {
+  editor: SourceEditor;
+
+  componentDidMount() {
+    this.setupEditor();
+  }
+
+  componentWillUnmount() {
+    this.editor.destroy();
+  }
+
+  shouldComponentUpdate(nextProps: Props) {
+    // This is needed, IMO, so we don't need to create
+    // loads of CM instances for no reason
+    return this.props.breakpoint != nextProps.breakpoint;
+  }
+
+  setupEditor() {
+    const { breakpoint } = this.props;
+    this.editor = createEditor(breakpoint.text);
+
+    // disables the default search shortcuts
+    // $FlowIgnore
+    this.editor._initShortcuts = () => {};
+
+    const node = ReactDOM.findDOMNode(this);
+    if (node instanceof HTMLElement) {
+      const mountNode = node.querySelector(".breakpoint-label");
+      if (node instanceof HTMLElement) {
+        // $FlowIgnore
```

Comment:
I hated to use this but flow keeps yelling at me:  `Cannot assign empty string to mountNode.innerHTML because property innerHTML is missing in null [1].`